detect_secrets/plugins/common/yaml_file_parser.py

```diff
@@ -92,6 +92,11 @@ def _tag_dict_values(self, map_node):
                         str(value.__line__),
                         'tag:yaml.org,2002:int',
                     ),
+                    self._create_key_value_pair_for_mapping_node_value(
+                        '__is_binary__',
```

Comment:
I realize the other calls of this don't do this, but maybe we can use keyword args to make things clearer, it helps with the other function calls in this file.